test/software/amazon/ion/GoodIonTest.java

```diff
@@ -129,8 +125,13 @@ public void testIterateByteArray()
     /**
      * Test files containing values with unknown text for symbols.
      */
-    private static final String[] FILES_WITH_UNKNOWN_SYMBOL_TEXT =
-                                  { "good" + File.separator + "item1.10n", "good" + File.separator + "symbols.ion" };
+    private static final String[] FILES_WITH_UNKNOWN_SYMBOL_TEXT = {
+        "good" + File.separator + "item1.10n",
+        "good" + File.separator + "symbols.ion",
```

Comment:
good/symbols.ion was in `GoodIonTest` already, I only moved to add the new files. But you are right it doesn't need to be excluded there. 

Submitted a PR to fix the NOP ones, https://github.com/amzn/ion-tests/pull/34